README.md

```diff
@@ -1 +1,37 @@
-# software-registry
\ No newline at end of file
+# CU DBMI Software Registry
```

Comment:
Naming things is a challenge! Openly wondering if another name would provide additional elegance or sense of pride in the work for the department. Mostly I'm imagining members of DBMI saying something like "Hey I just got added to the \<name goes here\> today!" (what would add excitement / enhancement to this?). To this point, I think moving from "Software Registry" to "Wall of Software" is a great change (getting "registered" isn't often exciting and might be a negative).

Other things that come to mind: "DBMI Software Tapestry", "Software Hall of Fame", "Software Achievement Wall".  I can also see how being more humble / frank about it would be better 🙂 (feel free to dismiss this as overly-semantic).

Reply:
Yeah I think WOS is more descriptive and more fun. Tapestry would also be cool. Wouldn't wan't to call it HOF just b/c if you're package isn't in there it may feel bad.